### PR TITLE
INI cleanup.

### PR DESCRIPTION
**Pull request by AmbientMalice**
Removed redundant async FB copies, and disabled depth buffer copies on Shadow Man to fix pause menu.

### CODE REVIEW

File: ini/GLideN64.custom.ini [hunk 1]
```diff
@@ -136,8 +125,6 @@ frameBufferEmulation\copyToRDRAM=1
 
 [F-ZERO%20X]
 Good_Name=F-Zero X (E)(J)(U)
-frameBufferEmulation\copyToRDRAM=2
-
```
Comment (loganmc10) [inline ini/GLideN64.custom.ini]: This will make the spacing a little ugly

**Comment by AmbientMalice** [comment]
I considered removing sync FB copies for DK64, but I dunno. Basically, they have a minor HUD side effect, just like Perfect Dark and Quake II, but I figure most players are going to prefer performance.

**Comment by gonetz** [comment]
Please remove all empty entries with no options set.

**Comment by AmbientMalice** [comment]
Okay. What about DK64? Should I leave sync FB forced? Async is quite a bit faster, and the side effects are generally minor.

**Comment by gonetz** [comment]
Ok, remove Dk64 as well.

**Comment by gonetz** [comment]
merged: d061ca044de1
Thanks!

**Comment by purplemarshmallow** [comment]
@AmbientMalice have you checked Donkey Kong 64 (E)?
This version crashes with async fb

**Pull request closed without merging**